test/karma/spec/corpuslisting.js

```diff
@@ -1,62 +1,67 @@
-/* eslint-disable
-    no-undef,
-*/
-window.c = console
-window._ = require("lodash")
-window.settings = {}
-require("configjs")
-const commonSettings = require("commonjs")
+window.c        = console;
+window._        = require(`lodash`);
+window.settings = {};
+
+require(`configjs`);
+
+const commonSettings = require(`commonjs`);
+
 _.map(commonSettings, function(v, k) {
   if (k in window) {
-    console.error(`warning, overwriting setting${k}`)
-}
-  window[k] = v
-})
+    console.error(`warning, overwriting setting${k}`);
+  }
+  window[k] = v;
+});
+
+require(`../../../app/scripts/util.js`);
+require(`defaultmode`);
+
+describe(`config file`, function() {
+
+  it(`all corpora definitions have the all the required fields`, function() {
```

Comment:
If the poor grammar of “it all corpora” bothers you, there’s an alias called `specify`
```suggestion
  specify(`all corpora definitions have all the required fields`, function() {
```

Reply:
It doesn't but maybe it should? 😅 I'll leave this as is since these are tests written by Spraakbanken and I don't currently have plans to change / expand them, but that's good to know! I'll make use of that going forward.